globals.d.ts

```diff
@@ -30,6 +30,7 @@ declare const _WORKER_PATH_LOCATION_: string;
 type Base64String = string;
 type PrivateKey<T> = T;
 type SigningPublicKey<T> = T;
+type AugmentationFactor<T> = T;
```

Comment:
Does this need to be generic? Are there times when we represent this type as anything other than a `Uint8Array`?